src/bin/opensmt.cc

```diff
@@ -238,4 +210,80 @@ void catcher( int sig )
   }
 }
 
+
+void printHelp( )
+{
+    const char help_string[]
+        = "Usage: ./opensmt [OPTION] filename\n"
+          "where OPTION can be\n"
+          "  --help [-h]              print this help message and exits\n"
+          "  --version                prints opensmt version and exits\n"
+          "  -d                       executes dry run of opensmt\n"
+          "  -r <seed>                sets random seed to specific number\n"
+          "  -i                       produces interpolant in case of unsat\n"
+          "  -v                       verbose run of the solver\n"
+          "  -p                       for execution within a pipe\n"
+          "  --config=<filename>      use configuration file <filename>\n";
+    std::cerr << help_string;
+    exit( 0 );
+}
+
+SMTConfig parseCMDLine( int argc, char * argv[ ] )
+{
+    SMTConfig res;
+    int version = 0;
```

Comment:
I think it should not be a constant, since it is a part of options struct, and it is changed if version is actually present in the command

Reply:
I see. From my understanding of the man page, this flag is set if the long option is used but not if the short version is available as well. Hence I would use `versionOptFlag` identifier and would not use `dryRunOpt` because it should not be necessary since `-d` is available as well.

Reply:
It can be done without `dryRunOpt` variable in the current implementation of `parseCMDLineArgs`

Reply:
I would like to use a clearer identifier such as `versionOptFlag`. I think it is very unclear what `versionOpt` means without the knowledge of the `option` structure.

Reply:
Done